pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -739,11 +739,31 @@ func (ds *Plugin) openDB(cfg *configuration, isReadOnly bool) (*gorm.DB, string,
 			db.Close()
 			return nil, "", false, nil, err
 		}
+		// TODO: we should keep this logic for a minor release cycle to make sure stale entries are removed eventually.
+		// Delete this function in a future release.
```

Comment:
We should put the exact release we want to remove this in. Makes it easy to grep for. We should also open an issue for this.
```suggestion
		// TODO: we should keep this logic for a minor release cycle to make sure stale entries are removed eventually.
		// Remove in SPIRE 1.8.0
```

Reply:
I'll open the issue.